apps/api/src/app/widgets/usecases/get-notifications-feed/get-notifications-feed.usecase.ts

```diff
@@ -23,30 +26,41 @@ export class GetNotificationsFeed {
           '. Make sure to create a subscriber before fetching the feed.'
       );
     }
+
     const feed = await this.messageRepository.findBySubscriberChannel(
       command.environmentId,
       subscriber._id,
       ChannelTypeEnum.IN_APP,
       { feedId: command.feedId, seen: command.seen },
       {
-        limit: 10,
-        skip: command.page * 10,
+        limit: LIMIT,
+        skip: command.page * LIMIT,
       }
     );
 
-    let sampleFeedItem: MessageEntity;
     if (feed.length) {
-      sampleFeedItem = feed[0];
-    }
-
-    if (sampleFeedItem) {
       this.analyticsService.track('Fetch Feed - [Notification Center]', command.organizationId, {
-        _subscriber: sampleFeedItem._subscriberId,
+        _subscriber: feed[0]._subscriberId,
```

Comment:
@jainpawan21 this will fail here in case feed is empty. I would actually prefer the `sampleFeedItem` safe guard in the if statement. or doing:
```suggestion
        _subscriber: feed[0]?._subscriberId,
```